data-raw/standard-format-data-prep/mark_recapture_standard_format.Rmd

```diff
@@ -170,7 +170,7 @@ butte_released <- read_csv(here::here("data", "rst", "butte_releases.csv")) %>%
          run_released = ifelse(is.na(run_released), "not recorded", tolower(run_released)),
```

Comment:
In the butte_recaptured table can we make the same change to "okie dam fyke trap"? I think right now it is still just "okie fyke trap"

Reply:
updated